examples/dti_prediction/model.py

```diff
@@ -0,0 +1,78 @@
+import pytorch_lightning as pl
+import torch
+from torch.nn import functional as F
+
+from kale.embed.deepdta import DeepDTAEncoder, MLPDecoder
+
+
+class LitDeepDTA(pl.LightningModule):
+    def __init__(self, num_drug_embeddings, drug_dim, drug_length, num_filters, drug_filter_length,
+                 num_target_embeddings, target_dim, target_length, target_filter_length, decoder_in_dim,
+                 decoder_hidden_dim, decoder_out_dim, dropout):
+        super().__init__()
+        self.drug_encoder = DeepDTAEncoder(num_embeddings=num_drug_embeddings, embedding_dim=drug_dim,
+                                           sequence_length=drug_length, num_kernels=num_filters,
+                                           kernel_length=drug_filter_length)
+
+        self.target_encoder = DeepDTAEncoder(num_embeddings=num_target_embeddings, embedding_dim=target_dim,
+                                             sequence_length=target_length, num_kernels=num_filters,
+                                             kernel_length=target_filter_length)
+        self.decoder = MLPDecoder(in_dim=decoder_in_dim, hidden_dim=decoder_hidden_dim,
+                                  out_dim=decoder_out_dim, dropout=dropout)
+
+    def forward(self, x_d, x_t):
+        drug_emb = self.drug_encoder(x_d)
+        target_emb = self.target_encoder(x_t)
+        com_emb = torch.cat((drug_emb, target_emb), dim=1)
+        output = self.decoder(com_emb)
+        return output
+
+    def configure_optimizers(self):
```

Comment:
We used cfg for configure a lot so using cfg here should be clear and more compact.